app/helpers/movies_helper.rb

```diff
@@ -1,7 +1,13 @@
+# frozen_string_literal: true
```

Comment:
The bulk of the changes in the next 10 files will be like the one in this file: changing out the hardcoded url with the new service method.